src/pages/user/components/ProfileDetails.tsx

```diff
@@ -88,9 +89,12 @@ export const ProfileDetails = ({ user, onLogoutPressed }: Props) => {
             />
           )}
           {onLogoutPressed && (
-            <Button colorScheme="brand" rightIcon={<FaSignOutAlt />} onClick={() => onLogoutPressed()}>
-              Kijelentkezés
-            </Button>
+            <Stack direction={['column', 'column', 'row']} w={['100%', 'inherit']}>
+              <ReportModal />
+              <Button w={['100%', 'inherit']} colorScheme="brand" rightIcon={<FaSignOutAlt />} onClick={() => onLogoutPressed()}>
+                Kijelentkezés
+              </Button>
+            </Stack>
```

Comment:
With longer names it can look strange
![image](https://github.com/kir-dev/konzisite-frontend/assets/65909866/a742ed38-13fd-49c5-8b24-a0cd87f12000)